core/core.schema.go

```diff
@@ -41,6 +41,9 @@ type Core {
 
 	"Fetch a git repository"
 	git(remote: String!, ref: String): Filesystem!
+
+	"Push a multiplatform image"
+	pushMultiplatformImage(ref: String!, filesystems: [FSID!]!): Boolean!
```

Comment:
We don't have to polish this before merging... But I think we should polish it before adding it to the new core API. In other words let's not pass through this field as-is into the new core API. OK with you?

Reply:
Oh totally agree. I added this mostly to verify the underlying implementation will work. In the new API I think that platform versions should be bundled together into a single Directory (or Container, not sure yet). So then if you push a multiplatform directory/container it automatically becomes a multiplatform image, no separate `pushMultiplatformImage` needed.